### PR TITLE
feat: 🎸 add support for v8.2.x, v9.1.x, v10.2.x

### PR DESCRIPTION
**Pull request by NachoVazquez**
## PR Checklist

Please check if your PR fulfills the following requirements:

- [ x ] The commit message follows our guidelines: CONTRIBUTING.md#commit
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ x ] Docs have been added / updated (for bug fixes / features)

## PR Type

What kind of change does this PR introduce?

```
[ ] Bugfix
[ x ] Feature
[ ] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no api changes)
[ ] Build related changes
[ ] CI related changes
[ ] Documentation content changes
[ ] Other... Please describe:
```

## What is the current behavior?

Missing support for some minor version of Angular

Issue Number: 1

## What is the new behavior?

Adds supports for version 8.2.x, 9.1.x, 10.2.x of Angular.

## Does this PR introduce a breaking change?

```
[ ] Yes
[ x ] No
```

## Other information
Solves issues: #1

### CODE REVIEW

File: src/angular-versions.ts [hunk 1]
```diff
@@ -59,6 +59,64 @@ export const versions = new Map<string, PackageJsonVersion>([
...
+        '@angular/cli': '~8.3.0',
+        '@angular/compiler-cli': '~8.2.3',
+        '@types/node': '~8.9.4',
+        codelyzer: '^5.0.0',
+        'ts-node': '~7.0.0',
+        tslint: '~5.15.0',
+        typescript: '~3.5.3'
+      }
+    }
+  ],
```
Comment (LayZeeDK) [inline src/angular-versions.ts:119]: There is no Angular 8.3, only Angular CLI 8.3.
```suggestion
```

File: src/angular-versions.ts [hunk 2]
```diff
@@ -59,6 +59,64 @@ export const versions = new Map<string, PackageJsonVersion>([
...
+        '@angular/platform-browser-dynamic': '~8.2.0',
+        '@angular/router': '~8.2.0',
+        rxjs: '~6.4.0',
+        tslib: '^1.10.0',
+        'zone.js': '~0.9.1'
+      },
+      devDependencies: {
+        '@angular-devkit/build-angular': '~0.802.0',
+        '@angular-devkit/build-ng-packagr': '~0.802.0',
+        '@angular/cli': '~8.2.0',
```
Comment (LayZeeDK) [inline src/angular-versions.ts:81]: Let's use the latest compatible CLI and builders
```suggestion
        '@angular-devkit/build-angular': '~0.803.0',
        '@angular-devkit/build-ng-packagr': '~0.803.0',
        '@angular/cli': '~8.3.0',
```

File: README.md [hunk 3]
```diff
@@ -61,7 +61,10 @@ At the moment this actions supports a limited set of Angular versions listed bel
 
 - 8.0.x
 - 8.1.x
+- 8.2.x
+- 8.3.x
```
Comment (LayZeeDK) [inline README.md:65]: There is no Angular 8.3, only Angular CLI 8.3.
```suggestion
```

**Review by LayZeeDK — CHANGES_REQUESTED** [review]

**Review by LayZeeDK — APPROVED** [review]

**Comment by LayZeeDK** [comment]
How do we bump the version number?

**Comment by NachoVazquez** [comment]
> How do we bump the version number?

`yarn release`

One important thing to notice is that before a release is ready we have to build the actions and commit the dist files. I know it sounds weird but it is the way it is done because Github uses those files directly from the repo to deploy the action.

This can be useful

https://github.com/actions/toolkit/blob/master/docs/action-versioning.md
https://docs.github.com/en/free-pro-team@latest/actions/creating-actions/creating-a-javascript-action

**Review by LayZeeDK — APPROVED** [review]

**Pull request merged**